Flagsmith-engine/Utils/Hashing.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+namespace Flagsmith_engine.Utils
+{
+    internal static class Hashing
+    {
+        public static float GetHashedPercentageForObjectIds(List<string> objectIds, int iteration = 1)
+        {
+            var toHash = String.Join(",", repeatIdsList(objectIds, iteration));
+            var hashedValueAsInt = BitConverter.ToInt64(CreateMD5(toHash),0);
+            var SS = hashedValueAsInt % 9999;
+            float SSS = SS / 9998;
```

Comment:
Excuse my Dot net skills, but I am Not sure what is going on here? Why is `SS` and `SSS` defined?

Reply:
sorry these two lines mistakenly added .Thanks for the identifying, will be updating it.